samples/samples/test_autocommit.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import uuid
+
+from google.api_core.exceptions import DeadlineExceeded
+from google.cloud import spanner
+import pytest
+
+import autocommit
+
+
+def unique_instance_id():
+    """Creates a unique id for the database."""
+    return "test-instance-{}".format(uuid.uuid4().hex[:10])
+
+
+def unique_database_id():
+    """Creates a unique id for the database."""
+    return "test-db-{}".format(uuid.uuid4().hex[:10])
+
+
+INSTANCE_ID = unique_instance_id()
+DATABASE_ID = unique_database_id()
+
+
+@pytest.fixture(scope="module")
+def spanner_instance():
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(INSTANCE_ID)
+    op = instance.create()
+    op.result(120)  # block until completion
+    yield instance
+    instance.delete()
+
+
+@pytest.fixture(scope="module")
+def database(spanner_instance):
+    """Creates a temporary database that is removed after testing."""
+    db = spanner_instance.database(DATABASE_ID)
+    db.create()
+    yield db
+    db.drop()
+
+
+def test_enable_autocommit_mode(capsys):
```

Comment:
```suggestion
def test_enable_autocommit_mode(capsys, database):
```

Looks like `database` is unused? I get this error without this change:

```
...
>           raise ValueError("instance '%s' does not exist." % instance_id)
E           ValueError: instance 'test-instance-86c4b1fe62' does not exist.

google/cloud/spanner_dbapi/connection.py:262: ValueError
```